python-api/gstswitch/connection.py

```diff
@@ -151,6 +151,26 @@ def connect_dbus(self):
             new_message = "{1} ({0})".format(message, self.address)
             raise ConnectionError(new_message)
 
+    def signal_subscribe(self, signal_handler):
+        """Subscribe to Signals on the bus"""
+        if not hasattr(signal_handler, '__call__'):
+            raise ValueError('Provided signal_handler is not callable')
+
+        try:
+            self.connection.signal_subscribe(
+                None,  # sender
+                self.default_interface,
+                None,  # member
+                self.object_path,
+                None,  # arg0
+                Gio.DBusSignalFlags.NONE,
+                signal_handler,
+                None)  # user_data
+        except GLib.GError as error:
```

Comment:
Just a FYI, it's pretty normal to do a
```python
try:
  blah
except Glib.GError as e:
  blah
```

I believe it is one of the cases that pep8/pylint will access a single letter name.